cli/internal/mcp/instructions-platform-development.md

```
@@ -0,0 +1,423 @@
+    properties:
+      name: suga-vpc
+```
+
+## REQUIRED Workflow for Platform Development
+
+### Step 1: Discover Available Plugins
+
+Use MCP tools to find plugins to compose:
+
+1. **Call `list_plugin_libraries()`** to see available libraries
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)